docs/operations_guide/Common.md

```diff
@@ -159,40 +169,30 @@ Please note that:
 * Kafka lag is reported with some delay (offsets are not committed immediately). It’s normal to have temporary spikes.
 * Large differences between slots in lag may point to data skew or e.g. problematic data for some keys
 
-### Handling typical scenario errors
```

Comment:
We have  a mix of "handling errors" and "where to find logs" here; all this very thoroughly mixed up. Changed to separate sections:

- Monitoring - is my scenario doing well?
- Logs
-  Handling typical scenario errors